tests/ui/pages/desktop/search.py

```diff
@@ -0,0 +1,52 @@
+from time import strptime, mktime
+
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class SearchResultList(Base):
+    """Search page"""
+    _search_text_locator = (By.CSS_SELECTOR, '.primary > h1')
+
+    _results_locator = (By.CSS_SELECTOR, "div.items div.item.addon")
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda _: self._search_text_locator)
+        return self
+
+    @property
+    def results(self):
+        """List of results"""
+        elements = self.selenium.find_elements(*self._results_locator)
+        return [self.SearchResultItem(self, el) for el in elements]
+
+    def sort_by(self, type):
+        from pages.desktop.regions.sorter import Sorter
+        Sorter(self.selenium, self.base_url).sort_by(type)
+
+    class SearchResultItem(Region):
+        """Represents individual results on the search page."""
+        _root_locator = (By.CSS_SELECTOR, '.item')
+
+        _name_locator = (By.CSS_SELECTOR, 'h3 > a')
+        _sort_criteria_locator = (By.CSS_SELECTOR, '.vitals > .adu')
+        _rating_locator = (By.CSS_SELECTOR, '.rating > .stars')
+
+        @property
+        def name(self):
+            """Extension Name"""
+            return self.find_element(*self._name_locator).text
+
+        @property
+        def users(self):
+            """Extensions users"""
+            number = self.find_element(*self._sort_criteria_locator).text
+            return int(number.split()[0].replace(',', ''))
```

Comment:
I could also check if the term ```downloads``` is within the element found and raise an exception.